core/templates/dev/head/components/fallback_editor.html

```diff
@@ -0,0 +1,66 @@
+<script type="text/ng-template" id="components/fallbackEditor">
+  <div style="height: 100%;">
+
+    <div ng-if="!triggerEditorIsOpen"
+         style="height: 100%;"
+         ng-attr-title="<[isEditable ? 'Edit fallback condition' : '']>">
+      <div class="oppia-readonly-rule-tile protractor-test-edit-trigger-button"
+           ng-class="{'oppia-editable-section': isEditable}">
+        <div class="oppia-rule-preview oppia-transition-200">
+          <div class="oppia-click-to-start-editing protractor-test-open-trigger-editor"
+               ng-if="isEditable" ng-click="openTriggerEditor()">
+          </div>
+
+          Once the learner has made <[trigger.customization_args.num_submits.value]>
```

Comment:
It seems odd to hard-code this in here when we're going to have other fallbacks in future.